### DISCUSSION TITLE
How can i only run failed cases again after finish the fuzz test?

### DISCUSSION THREAD

**Discussion by kraylv** [jtpereyda/boofuzz#727] [Q&A]
When i finished all the cases in a fuzz test,i want to run the failed cases again to double check the result.How can i do it?
Please help!!!
Thank you a lot!!!

**Comment by cq674350529** [comment]
Hi, according to the following code, there are two parameters named `index_start`, `index_end` in the `Session` class, which can be used to replicate a single test case, or a range of test cases. This may be suitable for your case.

https://github.com/jtpereyda/boofuzz/blob/ec4dfb5f5ac2d170fdc9f9418e28544e563479a6/boofuzz/sessions/session.py#L49-L57

**Reply by kraylv** [reply]
Thank you for your answer and support!
I will try it if it works.
